app/Http/Requests/CircleRequest.php

```diff
@@ -64,6 +66,8 @@ public function rules()
             'default_opt_in' => 'integer|min:0|max:1',
             'discord_webhook' => 'url',
             'only_giver_vouch' => 'integer|min:0|max:1',
+            'min_vouches_percent' => 'numeric|min:1|max:100',
+            'calculate_vouching_percent' => 'integer|min:0|max:1',
```

Comment:
This is either 0 or 1? Should this just be a boolean? Maybe I'm not following...

Reply:
Oh, I see it's a bollean in the database, so this is just to receive it from the frontend, right?